### DISCUSSION TITLE
Edit hourly rate on project level

### DISCUSSION THREAD

**Discussion by infeeeee** [kimai/kimai#2928] [General]
Currently it's only possible to delete or add a hourly rate, it's not possible to change them on the project or activity levels. The current workflow is to delete and than recreate if a user's (or project's) hourly rate changes. This workflow is really error prone.

It would be nice, if the value could be changeable without deleting. I see on timesheets it's changeable one by one, but it's not changeable on the project level. This change shouldn't affect existing timesheets, only for the future (e.g. a user got a raise)

**Comment by kevinpapst** [comment]
A discussion is not a feature request 😄
